### ISSUE TITLE
Enhancement: Services Loaded

### ISSUE THREAD

**Issue by eljeffeg** [CybercentreCanada/assemblyline-ui-frontend#523]
It would be nice if you could add a column for Internal / External services.  That would be great to see at a glance since it involves less network isolation and sending info about your malware to others (even if it's just a hash).

**Comment by cccs-sgaron** [comment]
Could you provide more info about where exactly you would want this? Screenshot with a mock up maybe?

**Comment by eljeffeg** [comment]
![Mockup](https://user-images.githubusercontent.com/161904/217909531-09038b29-abd2-4ece-81d1-aa98fa718aad.png)

**Comment by eljeffeg** [comment]
I did notice that Category has an option for `External` and `Internet Connected` (seems redundant), but I rather that be a description of the tool function, such as Static Analysis.  I could see a `Hash Lookup` as a category since a few of those services, like Beaver, are could be either static or dynamic.  The function of the tool category is a hash lookup and pull down any results.

**Comment by cccs-sgaron** [comment]
You can define your own categories and change the services categories as you please from the config.yml and the web UI. 

As for External vs Internet connected, they means different things: Internet Connected is for things like URL downloader which fetches URL found during analysis from the internet were as External is a service running outside AL infrastructure where you are likely sending the file/hash you are scanning to.

**Comment by eljeffeg** [comment]
Yes, sorry - I did note that distinction when I was looking at the updates myself.  Internal / External would be the relevant piece - what is being sent out of your environment.  Connectivity for updates and such is expected and not what I was concerned with quickly identifying.

**Comment by eljeffeg** [comment]
As background, some of the malware sent to AssemblyLine could be Classified.  Want to be hyper-sensitive to what leaves the system when bringing online new services and quickly identify such services.

**Comment by cccs-sgaron** [comment]
> As background, some of the malware sent to AssemblyLine could be sensitive. Want to be cautious about what leaves the system when bringing online new services and quickly identify such.

Agreed, this is why this prompt is there when you submit samples: 
![image](https://user-images.githubusercontent.com/59840781/218122349-f4220906-42a7-4d2f-be48-0aca17d5dee1.png)

**Comment by eljeffeg** [comment]
You mentioned you can add categories via the web UI.  I can't seem to find that.  I did find it in the config.yml, but wasn't sure if I need to duplicate the list plus add `Hash Lookup` or if I just add `Hash Lookup`.  Also, `Internet Connected` wasn't on the config.yml version in the docs, so not sure if that was a new addition or something added by a service which got pulled in.

**Comment by eljeffeg** [comment]
With regard to the message presented to users, might want to consider having it customizable with category.  We would see a big difference in risk between sending a hash to virus total vs uploading a binary to the service.  Not even sure I would present a message for a hash lookup.
